src/migration/1625968879644-addCourseAndAssignmentsModels2.ts

```diff
@@ -0,0 +1,18 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
```

Comment:
How do we feel about deleting all of these migrations, and squish them down into 1 for the PR?

Alternatively you can copy all the Ups into the up function, all the downs into the downs, and rename. (you'll have to toss/ revert your local db for testing)